MyMovies/View Controllers/MyMoviesTableViewController.swift

```diff
@@ -7,40 +7,128 @@
 //
 
 import UIKit
+import CoreData
 
 class MyMoviesTableViewController: UITableViewController {
+    
+    //let movieController = MovieController()
```

Comment:
I see you had this here, you just have it commented out. Having it instantiated in the cellForRowAt is going to have each cell have its own instance of the model controller. You would most likely want to init it here where its commented out and then just pass this instance to each cell.